themes/scss/infowindow/cartodb-infowindow-default.scss

```diff
@@ -92,8 +96,42 @@
   background: #888 !important;
 }
 
-.is-header .CDB-infowindow-mediaTitle {
+.CDB-infowindow.is-header .CDB-infowindow-mediaTitle {
   bottom: 12px;
+  display: block;
+}
+
+.CDB-infowindow.is-fail .CDB-infowindow-header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+
+.CDB-infowindow-fail {
+  display: block;
+  color: rgb(243, 86, 60);
+  font-size: 9px;
+  text-align: center;
+  text-transform: uppercase;
+}
+.CDB-infowindow.has-title.has-items.has-header-image .CDB-infowindow-fail {
+  position: absolute;
+  left: 20px;
+  top: 20px;
+  text-align: left;
+}
+.CDB-infowindow.has-title .CDB-infowindow-fail {
+  position: absolute;
+  left: 0px;
```

Comment:
Properties should be ordered position, top, left, width, text-align<br>`0px` should be written without units as `0`